tests/unit_tests/test_unit.py

```
@@ -2554,6 +2558,109 @@ def test_invalid_n_term_mod(tiny_config, reverse):
     assert torch.equal(finished, expected_finished)
 
 
+def _nterm_beam(tokenizer, aa_tokens, max_peptide_len, device):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_nterm_beam`

(ANN202)